src/social-network-provider/twitter.com/utils/user.ts

```diff
@@ -0,0 +1,12 @@
+import { SocialNetworkWorkerAndUI } from '../../../social-network/shared'
+import { isNull } from 'lodash-es'
+
+export const usernameValidator: SocialNetworkWorkerAndUI['isValidUsername'] = (name: string) => {
```

Comment:
Any reference at twitter for this function?

Reply:
![image](https://user-images.githubusercontent.com/15797507/63634417-cbe2c880-c688-11e9-9f13-071cb9803fd1.png)

Reply:
Uh... I mean the `name policy` in the twitter.com. Where do you find this rule?

Reply:
Sure, check it out. https://help.twitter.com/en/managing-your-account/twitter-username-rules